packages/web-sdk/src/transports/otlp/payload/OtelPayload.ts

```diff
@@ -1,12 +1,13 @@
-import { deepEqual, InternalLogger, Meta, TransportItem, TransportItemType } from '@grafana/faro-core';
+import { deepEqual, InternalLogger, TransportItem, TransportItemType } from '@grafana/faro-core';
 
 import { initLogsTransform, LogsTransform } from './transform';
 import type { ResourceLog } from './transform';
+import type { ResourceMetas } from './transform/types';
 import type { OtelTransportPayload } from './types';
 
 interface ResourceLogsMetaMap {
   resourceLog: ResourceLog;
-  meta: Meta;
+  resourceMetas: ResourceMetas;
```

Comment:
nit-picking / suggestion - remove the plural from the type and property? (resourceMeta: ResourceMeta) (just mentioning because it was singular before!)